### DISCUSSION TITLE
Bubble dialogs - one timeline for different Characters - how?

### DISCUSSION THREAD

**Discussion by mun001** [dialogic-godot/dialogic#2416] [Q&A]
Hello! I am using Dialogic 2 + Godot 4.3.

So, I am trying to use bubble-style dialogs. And have one (maybe stupid) question, I cannot find an answer.. So I have one timeline with some text like "Hi dude", and a 10 Characters (also 10 different Node2D), and I want to start bubble dialogs for them depends on some event in the game.. But I realized that if I don't put exact Character in Timeline, this is not working.. Doing register_character with different Character name & Node2D depends on game event also is not working..

So yeah, I can make a 10 different timelines putting where 10 Character names, but I think here should be more easy solution..

Thanks in advance!

**Comment by neilowhite** [comment]
Hi there, I think we spoke on DIscord about this? Sorry if it wasn't you! But if it was, I just thought I'd link to it in case anyone else was looking for a similar answer. Hope it worked out for you.

https://discord.com/channels/628713677239091231/1215327840040394772
